### ISSUE TITLE
loss and map

### ISSUE THREAD

**Issue by world2025** [balancap/SSD-Tensorflow#391]
hello , thanks for your codes , i have a question is that how i can get the loss per step in train  and how i can get the map in eval ,thank oyu
